client/src/Containers/Addquestion.jsx

```diff
@@ -148,7 +173,7 @@ const Addquestion = () => {
 
             <Button
               variant="contained"
-              href="/addtitle"
+              onClick={ () => setOpen(true)}

```

Comment:
It's not formatted correctly. Aren't you using `prettier` to format?

Reply:
How it should be formatted? `prettier` is not giving any error.